c99sh

```diff
@@ -193,13 +195,13 @@ done
 [ $v -ge 1 ] && set -x
 case "${name:0:3}" in
   c11|C11)
-    "${CC-cc}"   -std=gnu11   -o "$x" "${flags[@]}" -x c   "${logic[@]}" 1>&2
+    "${CC-cc}"   -std=gnu11   -o "$x" "${flags[@]}" -x c   "${logic[@]}" ${CF[@]} 1>&2
```

Comment:
Should this have double quotes around it?

Reply:
I originally did that, but found that multiple flags would get quoted in the final shell command. So if you did `c99sh -f "-O0 -fsummon-cthulu"` then cc would get `-O0\ -fsummon-cthulu`, so one long flag with escaped spaces. I guess it's got something to do with the processing that `$logic` and `$flags` receive that `$CF` doesn't. Anyways it's working correctly without quotes, but I don't know sh well so if you see an improvement let me know :)